docs/docs/migrations.md

```diff
@@ -306,6 +306,74 @@ queryInterface.removeIndex('Person', 'SuperDuperIndex')
 queryInterface.removeIndex('Person', ['firstname', 'lastname'])
 ```
 
+### addConstraint(tableName, indexNameOrAttributes, options)
```

Comment:
Why this is named as `indexNameOrAttributes`, I think it should be just `attributes`. Please explain your rationale behind this naming @harshithkashyap , I may be missing something